secure/key-manager/src/tests.rs

```diff
@@ -1,10 +1,14 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::{Action, Error, KeyManager, LibraInterface};
+use crate::{libra_interface::JsonRpcLibraInterface, Action, Error, KeyManager, LibraInterface};
+use anyhow::Result;
 use executor::{db_bootstrapper, BlockExecutor, Executor};
-use libra_config::{config::NodeConfig, utils::get_genesis_txn};
+use futures::{channel::mpsc::channel, StreamExt};
+use libra_config::{config::NodeConfig, utils, utils::get_genesis_txn};
 use libra_crypto::{ed25519::Ed25519PrivateKey, HashValue, PrivateKey, Uniform};
+use libra_json_rpc::bootstrap;
```

Comment:
don't import functions I think it is in the rust book, just call libra_json_rpc::bootstrap(...) inline